README.md

```diff
@@ -22,7 +22,7 @@ The MongoDB supported driver for Go.
 -------------------------
 ## Requirements
 
-- Go 1.10 or higher if using the driver as a dependency. Go 1.18 or higher if building the driver yourself. We aim to support the latest versions of Go.
+- Go 1.13 or higher if using the driver as a dependency. Go 1.18 or higher if building the driver yourself. We aim to support the latest versions of Go.
```

Comment:
The wording "building the driver yourself" is a bit misleading since most Go binaries are built from module source code. Consider separating that note as a sub-bullet and clarify that Go 1.18 is required to run the test suite.

```suggestion
- Go 1.13 or higher. We aim to support the latest versions of Go.
  - Go 1.18 or higher is required to run the driver test suite.
```